tests/cleanup_after_each.js

```diff
@@ -12,6 +13,7 @@ module.exports = function cleanupAfterEach() {
     nock.cleanAll()
     nock.enableNetConnect()
     nock.activate()
+    sinon.restore()
```

Comment:
This seems fine, though there is a call to `sinon.restore()` in `test_logging`. Would you mind removing that?